### PR TITLE
internal/cli: use config file for flags in new-cli, builder/files: update defaults

### PR DESCRIPTION
**Pull request by pratikpatil024**
This PR does the following changes

1. Adds support for toml configuration files.
2. Removed support for multiple configuration files.
3. Adds a new sub command in the new-cli named `dumpconfig` which exports the flags provided to it while running in a toml format (for now). This commands works in the exact same way as in geth. Usage: `bor dumpconfig <flags>`. 
4. Use string splice and trim function from old cli for better results. 
5. Updates the bor.service file and moved the flags to the config.toml file.
6. Updated the merge logic in `command.go`, and restructured the `Run()` function by creating a new function to extract values from the argument.

TODO's before merging
- [x] Discuss on the tag names for flags
- [x] Fix testcase (config_legacy_test.go)
- [x] Handel decoding of `*big.Int` type fields (`MaxPrice`, `IgnorePrice`, and `GasPrice`), and have a look at `time.Duration` type fields (`TxPool.LifeTime`, `TxPool.Rejournal`, and `Cache.Rejournal`) (It's working fine, but we still need to handle the `dumpconfig` output)
- [x] Test the working of -config flag
- [x] Unit test for flag parsing of each type (change assert to require).

Jira tickets:
[POS-586](https://polygon.atlassian.net/browse/POS-586): Move flags from CLI to config file

[POS-651](https://polygon.atlassian.net/browse/POS-651): Provide a default config file

[POS-635](https://polygon.atlassian.net/browse/POS-635): Add dumpconfig in bor

### CODE REVIEW

**Review by JekaMas — APPROVED** [review]

File: builder/files/bor.service [hunk 1]
```diff
@@ -6,34 +6,7 @@
...
-    # Uncomment and configure the following lines in case you run a validator. Don't forget to add backslash (\) 
-    # to previous command line.
-    # --keystore /var/lib/bor/keystore \
-    # --unlock [VALIDATOR ADDRESS] \
-    # --password /var/lib/bor/password.txt \
-    # --allow-insecure-unlock \
-    # --nodiscover --maxpeers 1 \
-    # --miner.etherbase [VALIDATOR ADDRESS] \
-    # --mine
+  ExecStart=/usr/local/bin/bor server -config="./config.toml"
```
[the remark this answers is not in the document]
Comment (pratikpatil024) [reply builder/files/bor.service:9]: Ohh, right. Will update it.

File: builder/files/config.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+chain = "mumbai"
```
[the remark this answers is not in the document]
Comment (pratikpatil024) [reply builder/files/config.toml:1]: Sure.

**Review by cffls — APPROVED** [review]
LGTM

**Review by JekaMas — APPROVED** [review]

**Review by manav2401 — APPROVED** [review]
Looks good. Great work 🚀

**Pull request merged**